src/pipelines/hospitalizations/gb_authority.py

```diff
@@ -21,44 +21,6 @@
 from uk_covid19 import Cov19API
 
 
-class ScotlandDataSource(DataSource):
```

Comment:
It looks like this was getting L3 information, but the replacement is L2.

Reply:
> @l2edzl3oy I think that you are replacing an L3 source with L1/L2, perhaps we can append the new sources instead? Do you know if the API provides hospitalization for L3?

So sorry @themonk911 ! The data replacement as a result of this code change is intended specifically for the `GB_SCT` key (and only that, not other `GB_SCT_*` keys one level below). I will undo the deletion.

Expected end state is to leave the `GB_SCT_*` (L3) data untouched, but override the `GB_SCT` (L2) data with the data from the UK govt data source. Per the [documentation](https://github.com/GoogleCloudPlatform/covid-19-open-data/blob/main/src/README.md#data-pipeline), this means that I should leave the current Scotland data source in `config.yaml` untouched, and only append the new UK L2 data source **after** the Scotland data source. The expected end state is then achieved. Is my understanding right?

As for your 2nd question - sadly I don't think the API provides hospitalization data for L3.

Reply:
^That sounds good to me - shame the API doesn't do L3. If you are interested in a another PR, you can look at the change in the excel parsing for the L3 data for GB_SCT_* :)

Reply:
Ok I've edited as suggested. Thanks for the heads up - will take a look and see if there are alternative data sources for that too.